lib/DependencyScan/DependencyScanningTool.cpp

```diff
@@ -162,7 +162,10 @@ DependencyScanningTool::initCompilerInstanceForScan(
   }
 
   // Setup the instance
-  Instance->setup(Invocation);
+  std::string InstanceSetupError;
+  if (Instance->setup(Invocation, InstanceSetupError)) {
+    return std::make_error_code(std::errc::not_supported);
+  }
```

Comment:
@artemcm Mind double checking it's correct we now bail here?

Reply:
Yeah, that looks correct to me.